CoreGraphFramework/src/au/gov/asd/tac/constellation/graph/schema/type/SchemaTransactionType.java

```diff
@@ -268,15 +268,24 @@ public Builder setIncomplete(final boolean incomplete) {
         }
 
         public SchemaTransactionType build() {
+            if (color == null) {
+                color = superType != null ? superType.color : UNKNOWN.color;
+            }
```

Comment:
not sure this is the correct fix. Previously it wasn't actually changing the value of `color`, just using the value to determine what to pass through. Similar for `style` and `directed`